style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  box-sizing: none;
+  margin: 0 auto;
+  padding: 0;
+}
+h2 {
+  font-size: 30px;
+}
+p {
+  font-size: 18px;
+}
+#main {
+  width: 1440px;
+  height: auto;
+  background-color: whitesmoke;
+  margin: 0 auto;
+}
+#title {
+  width: 100%;
+  height: 150px;
+  padding-top: 50px;
+  text-align: center;
+}
```

Comment:
Kindly I recommend you to use more classes instead of ids, because same class can be selected for more than one element, but same id can be selected and used for only one element.